pkg/ipsec/ipsec_vici_parse.go

```diff
@@ -416,15 +416,15 @@ func parseCertificate(cert *listCertParams) (*pb.ListCert, error) {
 		s1 := strings.ToUpper(cert.Type)
 		switch {
 		case strings.Contains(s1, "X509_AC"):
-			ret.Type = pb.CertificateType_CERT_X509
+			ret.Type = pb.CertificateType_CERTIFICATE_TYPE_X509_AC
 		case strings.Contains(s1, "X509_CRL"):
-			ret.Type = pb.CertificateType_CERT_X509_CRL
+			ret.Type = pb.CertificateType_CERTIFICATE_TYPE_X509_CRL
 		case strings.Contains(s1, "X509"):
-			ret.Type = pb.CertificateType_CERT_X509
+			ret.Type = pb.CertificateType_CERTIFICATE_TYPE_X509_UNSPECIFIED
```

Comment:
@sandersms is this correct mapping according to [this](https://github.com/opiproject/opi-api/commit/668f00af0660e190cfd5a253bea34b768bcda0c9#diff-b99ec512294c6cd1be64b93090ae6f089ae7272b0db67eee1d7dcbd4a80d36d1) api change?
Should our api look like this instead, so we could use `CERTIFICATE_TYPE_X509` here?
```
CERTIFICATE_TYPE_UNSPECIFIED = 0;
CERTIFICATE_TYPE_X509 = 1;
CERTIFICATE_TYPE_X509_AC          = 2;
CERTIFICATE_TYPE_X509_CRL         = 3;
CERTIFICATE_TYPE_OCSP_RESPONSE    = 4;
CERTIFICATE_TYPE_PUBKEY           = 5;
```

Reply:
good catch @artek-koltun 
```
	CERT_X509          = 0;
	CERT_X509_AC       = 1;
	CERT_X509_CRL      = 2;
	CERT_OCSP_RESPONSE = 3;
	CERT_PUBKEY        = 4;
```
to 
```
CERTIFICATE_TYPE_UNSPECIFIED = 0;
CERTIFICATE_TYPE_X509 = 1;
CERTIFICATE_TYPE_X509_AC          = 2;
CERTIFICATE_TYPE_X509_CRL         = 3;
CERTIFICATE_TYPE_OCSP_RESPONSE    = 4;
CERTIFICATE_TYPE_PUBKEY           = 5;
```
instead of
```
	CERTIFICATE_TYPE_X509_UNSPECIFIED = 0;
	CERTIFICATE_TYPE_X509_AC          = 1;
	CERTIFICATE_TYPE_X509_CRL         = 2;
	CERTIFICATE_TYPE_OCSP_RESPONSE    = 3;
	CERTIFICATE_TYPE_PUBKEY           = 4;
```

Reply:
The issue may require a different fix that I am working now based on the test failure in the strongswan